content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -278,10 +278,11 @@ rules:
   verbs: ["update", "get"]
 ```
 
-{{< note >}}
-You cannot restrict `create` or `deletecollection` requests by resourceName. For `create`, this
-limitation is because the object name is not known at authorization time.
-{{< /note >}}
+{{< caution >}}
+You cannot restrict `create` or `deletecollection` requests by their resource name.
+For `create`, this limitation is because the name of the new object may not be known at authorization time.
+`list`(and `watch`)actually can be restricted by their resource name, and permits list(and `watch`) requests using a fieldSelector of `metadata.name=...` to match a single object (for example, `/api/v1/namespaces/$ns/configmaps?fieldSelector=metadata.name=foo`). This is useful when using an informer to list/watch a single object.
+{{< /caution >}}
```

Comment:
- I got it wrong. We don't need a `caution` because there isn't a snag like I thought. `note` was right.

- Not all readers will recognize “informer”, which is a client-side helper component. Clients in Golang often use informers and _some_ other clients have similar mechanisms, but not all. I would not mention informer here.

The detail to mention is that:
- if you restrict `list` and `watch` by `resourceName`, then the _only_ way that a client including (`kubectl`) can perform that `list` or `watch` is by specifying a field selector that matches on `metadata.name`.

Reply:
Update.